### PR TITLE
CVR-9 return an HSTS header on every reply

### PR DESCRIPTION
**Pull request by briskt**
https://itse.youtrack.cloud/issue/CVR-9

### CODE REVIEW

File: application/domain/domain.go [hunk 1]
```diff
@@ -133,6 +133,7 @@ var Env struct {
 	ApiBaseURL                 string `required:"true" split_words:"true"`
 	AccessTokenLifetimeSeconds int    `default:"1166400" split_words:"true"` // 13.5 days
 	AppName                    string `default:"Cover" split_words:"true"`
+	HstsTtl                    int    `default:"3600" split_words:"true"` // default = 1 hour
```
Comment (Baggerone) [inline application/domain/domain.go:136]: When I search Google for "what's a good default value for Strict-Transport-Security max-age", I see

`It is advisable to assign the max-age directive's value to be greater than 10368000 seconds (120 days) and ideally to 31536000 (one year).`

**Review by Baggerone — APPROVED** [review]

File: application/domain/domain.go (hunk 1 above)
Comment (jason-jackson) [inline application/domain/domain.go:136]: What does `Ttl` stand for?  Would `HstsMaxAge` be clearer?

**Review by jason-jackson — APPROVED** [review]

File: application/domain/domain.go (hunk 1 above)
Comment (briskt) [reply application/domain/domain.go:136]: I think the advice from @devon-sil was to start low and work up after you're more confident that nothing is going to break.

Comment (briskt) [reply application/domain/domain.go]: "Time To Live". I had thought someone called it that, but looking back at the docs there is no mention of that term in this context.

**Pull request merged**